tests/test_utils/test_hdfs_mods.py

```diff
@@ -95,15 +103,42 @@ def expout_data(self):
     def test_hdfs_load_json(self, mock_hdfs, mock_json):
         """Test the expected functionality of hdfs_load_json."""
 
-        mock_f = mock.Mock()
-        mock_hdfs.open.return_value.__enter__.return_value = mock_f
+        mock_hdfs.open.return_value.__enter__.return_value = sys.modules["mock_f"]
 
         mock_json.load.return_value = self.input_data()
 
         json_result = hdfs_load_json("file/path/filename.json")
 
-        mock_json.load.assert_called_with(mock_f)
+        mock_json.load.assert_called_with(sys.modules["mock_f"])
 
         json_expout = self.expout_data()
 
         assert json_result == json_expout
+
+
+class TestCheckFileExists:
```

Comment:
This test passes but doesn't call the function with the mock `sys.modules["mock_f"]. While this test works if the file is a local one, it would fail when the file is on DAP/DevTest. You can still merge since it passes all tests but might need further revision

Reply:
https://jira.ons.gov.uk/browse/RDRP-246